documentation/src/main/asciidoc/topics/transactions.adoc

```diff
@@ -76,15 +75,11 @@ builder.transaction()
     .recovery()
     .enabled(false)
     .recoveryInfoCacheName("__recoveryInfoCacheName__");
-builder.versioning()
-    .enabled(false)
-    .scheme(VersioningScheme.NONE);
```

Comment:
@pruivo can you please take a look at these changes? I think we need to update to remove write-skew configuration and versioning. pretty sure I got most of the required updates.

Reply:
you can remove the `<versioning>` from `documentation/src/main/asciidoc/topics/config_examples/transaction_config.xml`

Reply:
done. thanks @pruivo